example/relational/README.md

```diff
@@ -260,6 +269,10 @@ inverse := Keyword{HashKey: article, SortKey: keyword, /* ... */}
 db.Put(forward)
 db.Put(inverse)
 ```
+=> would `forward` and `inverse` have both the content of the article or only one?
```

Comment:
Both but this is sole depends on the app design. E.g. here https://github.com/fogfish/dynamo/blob/7c624aed19be027d2e20cb182b2fdd1d1cbf3687/example/relational/types.go#L167 we just write references but often best practice say write whole data. There should be some balance between usability, minimal number of I/O to db and cost.

for example, if we would write the entire metadata about the article along with the keyword then we would need to update all these metadata when author changes the article name. We need to compromise.